op-e2e/inbox/inbox_test.go

```diff
@@ -0,0 +1,149 @@
+package inbox
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	batcherFlags "github.com/ethereum-optimism/optimism/op-batcher/flags"
+	op_e2e "github.com/ethereum-optimism/optimism/op-e2e"
+	"github.com/ethereum-optimism/optimism/op-e2e/bindings"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/wait"
+	"github.com/ethereum-optimism/optimism/op-e2e/system/e2esys"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	ctx, _ = context.WithTimeout(context.Background(), 20*time.Second)
+	cost   = big.NewInt(1500000000000000)
+)
+
+func TestBatchInboxFunctionSuccess(t *testing.T) {
+	op_e2e.InitParallel(t)
+
+	sys, l1Client := startSystemWithBatchInboxContract(t)
+	t.Cleanup(sys.Close)
+
+	// Wait for batch submitted and check event
+	requireEventualBatcherTx(t, &sys.Cfg, l1Client, 12*time.Second)
```

Comment:
Is there any particular reason to choose 12 seconds as the timeout parameter?

Reply:
update to 8 * time.Second.